src/args.rs

```diff
@@ -1,12 +1,11 @@
+use rusoto_core::Region;
+
 /// This tool does bulk operations against sub-components in a cluster. Use with great care.
 #[derive(Debug, StructOpt)]
 pub struct Args {
     /// AWS profile for authentication
     #[structopt(long = "profile")]
     pub profile: Option<String>,
-    /// AWS region of cluster
```

Comment:
These comments get converted into help text for the CLI, we should avoid removing them

Reply:
oh my bad want me to add that back now?

Reply:
please, for each of the new region parameters

Reply:
looks good, will merge after checks